notebooks/AFR-22/worksheet2.ipynb

```diff
@@ -739,7 +739,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.10"
```

Comment:
The python version installed by default by the conda yml is 3.6.10, do we need to get this updated?